composer.json

```diff
@@ -32,9 +32,10 @@
         "issues": "http://github.com/heyday/silverstripe-elastica/issues"
     },
     "require": {
-        "php": ">=5.6.0",
-        "ruflin/elastica": "~5.3",
-        "symbiote/silverstripe-queuedjobs": "~4.0"
+        "php": ">=7.2",
+        "ruflin/elastica": "^7",
+        "elasticsearch/elasticsearch": "~7.3.0",
```

Comment:
Is this strictly necessary? What does it give us that `ruflin/elastica` doesn't - and vice-versa?

Reply:
That might just be the latest version from when I made the changes a year ago lol.

Reply:
I can't remember exactly, but there was some versioning issue with the sub-dependency I had to adjust a few times. It was a year ago. :D